### PR TITLE
Create app skeleton

### PR DESCRIPTION
**Pull request by dcwds**
The following changes have been made according to the acceptance criteria:
- install fonts with `@font-face`
- reset default browser styles with `normalize.css`
- add global styles
- add router with routes `/` and `/search`
- changes to folder structure

### CODE REVIEW

**Comment by mentor-tara** [comment]
Great job :rocket: I'm a bit busy at the moment but I'll leave a review soon.

In the meantime **I'd suggest you start with the next task already. Even if you just check out a new branch** where you can continue to work on later. This is a great opportunity to learn professional Git workflows. 

[Head back to the course material to learn how to proceed.](https://profy.dev/project/reddit-timer/while-waiting)

PS: Of course we all know I'm not busy. I'm a bot :robot: But let's pretend that I am to have a more realistic team environment. We're here to practice after all, right? :smile:[](ckkp04mmw2vuk0b42idpakyah)

**Review by mentor-tara — CHANGES_REQUESTED** [review]
Before I approve this PR I'd like to share a few tips with you.

**Please go through the list below and review your own code first.** Imagine you're the reviewer or a person who didn't see the code before. Ask yourself if everything is clearly written and understandable. Are components and variables properly named?

Reviewing your own code with this mindset is an important practice that will help you learn a lot and write high-quality code.

To make it a bit easier for you here are the tips I promised. Some have a link :link: attached where you can find more information. I'd recommend reading the **bold tips** in any case. Feel free to check the boxes when you think your code fits.

**General:**
- [ ] **Use yarn instead of npm** [:link:](https://profy.dev/project/reddit-timer/review-tips/use-yarn)
- [ ] **Use a common project structure** [:link:](https://profy.dev/project/reddit-timer/review-tips/file-structure)

Once you're done with reviewing your own code feel free to check the [example implementation](https://profy.dev/project/reddit-timer/code-examples/create-app-skeleton-d137af0d-ce79-4289-ae9b-df6f0444a380). **But remember: It's an important exercise to review your code first on your own.**

When you think this PR is ready to be merged just let me know by requesting another review. I'll approve right away.[](ckkp0fo0g2w0w0b422db33fjd)

File: src/components/app/app.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+const App = () => (
+  <Router>
+    <Switch>
+      <Route exact path="/">
+        <div>
+          <code>/</code>
+          {' '}
+          placeholder
+        </div>
```
Comment (jkettmann) [inline src/components/app/app.js:14]: Your PR looks good. I like that you wrote a detailed description 🚀 

Just FYI: This code here and the placeholder for the search bar could be shorter just to remove some clutter. You can simply write 

```
<Route exact path="/">
  Home
</Route>
```

Comment (dcwds) [reply src/components/app/app.js:14]: Hey, thanks!

Yeah, it's strange. I believe ESLint is the culprit for the whitespace insertions in the App component. I ran `yarn format`, which doesn't bother removing the whitespace. I had disabled Prettier (which I have configured globally in VSCode) as well.

In any case, I will opt for your simpler approach with the `Home` placeholder. I also intend to go with your `styled-components` strategy for managing global styles.

I noticed in your linked example that the semi-bold fonts are included, yet in the marketing site mock-up, there appears to only be regular weights. I suspect the Reddit Timer app (that's included in premium) uses them.

Again, thanks for taking a look at everything! I really appreciate it.

**Comment by dcwds** [comment]
The following changes have been made:
- simplify route placeholders
- discard `normalize.css` package and use `styled-normalize`
- discard global css and use `createGlobalStyle` (via `styled-components`) to handle global css
- move fonts to `/public` dir

**Review by mentor-tara — APPROVED** [review]
Great work! :tada::tada::tada: You can merge the PR now.[](ckkqnj5io3o540b425ilxp80y)

File: src/components/app/app.js (hunk 1 above)
Comment (jkettmann) [reply src/components/app/app.js]: You're welcome. About the fonts: I'm not sure tbh. Maybe they are used on the search page. Maybe it's a left-over from an earlier implementation. I cleaned up the designs at some point. Maybe I forgot to change the example implementation 😉

**Pull request merged**